src/v0/destinations/google_adwords_enhanced_conversions/networkHandler.js

```diff
@@ -90,23 +106,41 @@ const getConversionActionId = async (method, headers, params) => {
  * @returns
  */
 const ProxyRequest = async (request) => {
-  const { body, method, endpoint, params } = request;
+  const { body, method, endpoint, params, metadata } = request;
   const { headers } = request;
 
-  const conversionActionId = await getConversionActionId(method, headers, params);
+  const conversionActionId = await getConversionActionId({ method, headers, params, metadata });
 
   set(
     body.JSON,
     'conversionAdjustments[0].conversionAction',
     `customers/${params.customerId}/conversionActions/${conversionActionId}`,
   );
+  logger.requestLog(`[${destType.toUpperCase()}] conversion enhancement request`, {
+    metadata,
+    requestDetails: { url: endpoint, body: body.JSON, method },
```

Comment:
Don't we need to capture headers of request ?

Reply:
Request headers usually contain `Authorization` or some kind of headers used by destination for authentication/authorization. Logging such data could be risky for us. This is the thought process behind proceeding with not logging request headers

Although we can start with masking `Authorization` header alone & see if some implementation at destination level would be required. Let me know if this approach is to be taken.

Reply:
valid but they would be other non-sensitive information which would be needed to log.
This should be done case-by-case basis on what headers needed to be logged at destination level. But still to prevent any misuse of sending entire headers object, we need fall back for masking sensitive keys identified in the utility - can take this as separate task

Reply:
I will add this to the logger improvements task that we had discussed about earlier